resources/views/committee/members.blade.php

```diff
@@ -10,8 +10,20 @@
 
             <div class="panel-body">
                 <p>
+                    <b><u>Columns</u></b><br>
+                    <div id="columnControls">
```

Comment:
You might want to use a `<label><input type='checkbox' name='crsid' checked>CRSid</label>` tag so that when you click on the text, it triggers the checkbox as well (:

I do believe that with HTML5, you can and should use `checked` instead of `checked='checked'`.

You could look at replacing the `<span style="margin-right:1em">` repeated bits with having a css rule, e.g.
```
#columnControls > input {
    margin-right: 1em;
}
```

Or you might find that you don't need it by using the appropriate Bootstrap class, e.g. `checkbox-inline`. Check out http://getbootstrap.com/css/#inline-checkboxes-and-radios (: